src/components/topNav/TopNav.module.css

```diff
@@ -1,22 +1,52 @@
-.ul {
-  display: flex;
+.primary,
+.secondary {
+  width: fit-content;
 }
 
-.ul > * {
-  flex: 1;
+.ul {
+  margin: unset;
+  display: flex;
+  padding-inline-start: unset;
+  align-items: center;
 }
 
 .li {
   list-style-type: none;
-  padding-block-start: var(--conduction-navbar-padding);
-  padding-block-end: var(--conduction-navbar-padding);
 }
 
-.li > * {
+.li > a {
+  display: block;
+  padding-inline-start: var(--nlportal-space-inline-lg);
+  padding-inline-end: var(--nlportal-space-inline-lg);
+  padding-block-start: var(--nlportal-space-inline-lg);
+  padding-block-end: var(--nlportal-space-inline-lg);
+}
+
+.li > .link,
+.li > .anchor,
+.li > .anchor > .link {
+  color: var(--denhaag-color-white);
```

Comment:
I feel as if this shouldn't always be white; right now, when I implemented this on a white background, I thought it wasn't working.

Maybe it should be a prop? Or default to something else?

Reply:
@hghanbari I see that this is stil unresolved?